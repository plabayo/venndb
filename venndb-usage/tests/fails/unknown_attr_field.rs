use venndb::VennDB;

#[derive(Debug, VennDB)]
struct Employee {
    #[venndb(foo)]
    id: u32,
}

fn main() {}
