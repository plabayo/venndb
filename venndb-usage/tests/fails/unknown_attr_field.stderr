error: Invalid field-level `venndb` attribute
       Expected one of: `key`
 --> tests/fails/unknown_attr_field.rs:5:14
  |
5 |     #[venndb(foo)]
  |              ^^^
